code/modules/mod/mod_theme.dm

```diff
@@ -645,7 +644,7 @@
 		Heatsinks line the sides of the suit, and greater technology has been used in insulating it against \
 		both corrosive environments and sudden impacts to the user's joints."
 	default_skin = "safeguard"
-	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, 	BOMB = 40, BIO = 100, FIRE = 100, ACID = 95, WOUND = 25)
+	armor = list(MELEE = 15, BULLET = 20, LASER = 15, ENERGY = 15, BOMB = 40, BIO = 100, FIRE = 100, ACID = 95, WOUND = 15)
```

Comment:
considering this has greater fire protection, and, from what the description suggests, more speed, these armour values should not be greater than the normal security modsuits.

I know this is the HoS's modsuit, but that creates two modsuits which are objectively better than all of the station modsuits. I don't mind the HoS's modsuit being great, but it shouldn't be a straight up upgrade to all other modsuits (except the Captain's modsuit)

In my opinion, this should be slightly weaker than the security modsuit, but not by a whole lot so that players can take the security modsuit if they're willing to trade off fire protection and speed for slightly more armour, though I doubt it'd make a big difference, it'd just show to players that the HoS suit isn't just automatically better, and some tradeoffs need to be made, albeit not much of a tradeoff.